src/audio/pipeline/pipeline-stream.c

```diff
@@ -219,6 +227,10 @@ int pipeline_trigger(struct pipeline *p, struct comp_dev *host, int cmd)
 	if (data.cmd != cmd) {
 		list_init(&walk_ctx.pipelines);
 
+		if (data.delay)
+			wait_delay(clock_ms_to_ticks(PLATFORM_DEFAULT_CLOCK,
```

Comment:
We should not be busy waiting. We should do the early trigger on LL tick 0, trigger on LL tick(delay) and then copy() on LL tick(delay + 1)

Reply:
@lgirdwood this is for DMA scheduling pipelines. For timer-driven pipelines a later commit adds a `return` above that and then just skips the required number of scheduling ticks

Reply:
ok, need this in a comment.